docs/logs/correlation/Program.cs

```diff
@@ -34,8 +36,10 @@ public static void Main()
             logger.FoodPriceChanged("artichoke", 9.99);
         }
 
-        // Dispose logger factory before the application ends.
-        // This will flush the remaining logs and shutdown the logging pipeline.
+        // This will flush the remaining logs.
+        loggerProvider.ForceFlush();
```

Comment:
I don't think this is sufficient:
1. We should call out that the caller is responsible for checking the return value and act on it, and provide some guidance on how to handle the return value.
2. I don't think normal users should call ForceFlush without explicitly providing a timeout value.

Reply:
Attempted a version of this prose as well as with a timeout value in the `extending the sdk` project.